cmd/podman/images.go

```diff
@@ -159,6 +159,17 @@ func imagesCmd(c *cliconfig.ImagesValues) error {
 		filters = append(filters, fmt.Sprintf("reference=%s", image))
 	}
 
+	var sortValues = map[string]bool{
+		"id":         true,
+		"size":       true,
+		"tag":        true,
+		"repository": true,
+		"created":    true,
+	}
```

Comment:
nit, could we change the above to top down alpha order?  created, id, repository, size, tag

Reply:
Fixed

Reply:
ty!